### PR TITLE
Fixed limit controls in textstyle

### PR DESCRIPTION
**Pull request by JeffEverett**
Addresses issue #318.

Sorry, I should have caught this issue. I just didn't realize the limit controls were supposed to work in textstyle (probably because the operators that normally have limits don't in textstyle unless you explicitly specify that they should using `\limits`).

I updated the `LimitControls` screenshot test with the use of a `\limits` function in textstyle.

As an aside, MathML output still does not support limits in both dipslaystyle and textstyle (as documented in #198). I think I will tackle this issue soon to complete KaTeX's limit support.

### CODE REVIEW

File: src/Parser.js [hunk 1]
```diff
@@ -263,6 +263,7 @@ Parser.prototype.parseAtom = function(pos, mode) {
             else {
                 var limits = lex.text == "\\limits";
```
Comment (xymostech) [inline src/Parser.js]: Can you change this to `===` while you're here? :)

File: src/Parser.js [hunk 2]
```diff
@@ -263,6 +263,7 @@ Parser.prototype.parseAtom = function(pos, mode) {
...
                 base.result.value.limits = limits;
+                base.result.value.alwaysHandleSupSub = true;
```
Comment (xymostech) [inline src/Parser.js]: I'm a little sad that we're adding new properties here, but this is fine.

**Comment by xymostech** [comment]
:+1: Thanks for fixing this so quick! Just one little nit, then this looks great!

**Comment by JeffEverett** [comment]
@xymostech No problem. Just changed the `==` to `===`.

**Pull request merged**